### ISSUE TITLE
Replace hardcoded strings with _t()

### ISSUE THREAD

**Issue by Zauberfisch** [swipestripe/silverstripe-swipestripe#33]
I have seen some places where strings are hard coded into code or template.
To keep the system translatable we should make sure all strings use `_t()` or `<%t ... %>`.

**Comment by frankmullenger** [comment]
Absolutely, this would be great. I've tried to cover as many as I can but there are no doubt still gaps. Here is the transifex project: https://www.transifex.com/projects/p/swipestripe/

**Comment by Zauberfisch** [comment]
ok, haven't dealt with transifex yet, will have to look into that.
